tools/dbus_exporter.py

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env python3
 # SPDX-License-Identifier: LGPL-2.1-or-later
+import os
 from argparse import ArgumentParser
 from pathlib import Path
 from subprocess import PIPE, run
 
 
 def extract_interfaces_xml(output_dir, executable):
+    # If proc is not mounted, set LD_LIBRARY_PATH so that shared/core libs can be found
+    env = os.environ.copy()
+    if not os.path.exists('/proc/self'):
+        if 'LD_LIBRARY_PATH' not in env:
+            env["LD_LIBRARY_PATH"] = "src/shared:src/core"
+        else:
+            env["LD_LIBRARY_PATH"] = f"src/shared:src/core:{env['LD_LIBRARY_PATH']}"
```

Comment:
Setting LD_ORIGIN_PATH to the parent dir of executable should also work and feels a bit cleaner: https://github.com/bminor/glibc/blob/0242c9f9e606ade838651dadea13c251e3cc4ac2/sysdeps/unix/sysv/linux/dl-origin.c#L55C42-L55C56

Reply:
nice, it works